riemann/tests/helpers.py

```diff
@@ -724,23 +736,23 @@
             'expiry': bytes.fromhex('00000000'),
             'ins': [
                 {
-                    'id': 0, 
+                    'id': 0,
                     'hash': bytes.fromhex(''),
                     'index': bytes.fromhex(''),
                     'tree': bytes.fromhex(''),
                     'outpoint': bytes.fromhex(''),
                     'in': bytes.fromhex('04aacce7ca34e1f59e55d957f4d27aa6f54c5dd4046665840797ffe88b27320a0100000000ffffffff')
                     },
                 {
-                    'id': 1, 
+                    'id': 1,
                     'hash': bytes.fromhex(''),
                     'index': bytes.fromhex(''),
                     'tree': bytes.fromhex(''),
                     'outpoint': bytes.fromhex(''),
                     'in': bytes.fromhex('0785b51df7d46512ebd63c4dd17f391360c9d6fc5c8846a0684184a601c30c790100000000ffffffff')
                     },
                 {
-                    'id': 2, 
+                    'id': 2,
                     'hash': bytes.fromhex(''),
                     'index': bytes.fromhex(''),
                     'tree': bytes.fromhex(''),
```

Comment:
Should stack_items and script_sig be in witness in L807-816 be a part of the witness list starting on L781?